scalatest/src/main/scala/org/scalatest/Suite.scala

```diff
@@ -2010,7 +2010,7 @@ used for test events like succeeded/failed, etc.
   def autoTagClassAnnotations(tags: Map[String, Set[String]], theSuite: Suite) = {
     // SKIP-SCALATESTJS,NATIVE-START
     val suiteTags = for {
-      a <- theSuite.getClass.getAnnotations
+      a <- theSuite.getClass.getAnnotations.map(_.nn)
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.